.github/workflows/continuous-integration.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: "Build SDK"
         run: |
-          MVN_ARGS="${{ env.MVN_MULTI_THREADED_ARGS }} clean install -DskipTests -DskipFormatting"
+          MVN_ARGS="${{ env.MVN_MULTI_THREADED_ARGS }} clean install -Dgenerate -DskipTests -DskipFormatting"
```

Comment:
(Question)

Just to clarify, the generation happens with every commit of every PR right?

Reply:
Yes 👍🏻